### PR TITLE
Web ui

### PR DESCRIPTION
**Pull request by chetan253**

### CODE REVIEW

File: web-ui/src/components/Profile/profile.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
+import "./styles.css";
+
+class Profile extends Component{
+    render() {
+        return (
+            <div className="card UserProfile">
+                <img src="https://www.bsn.eu/wp-content/uploads/2016/12/user-icon-image-placeholder-300-grey.jpg" alt="User Placeholder" className="img-thumbnail rounded-circle"/>
```
Comment (bhushanmalgaonkar) [inline web-ui/src/components/Profile/profile.jsx]: I think it's better to download the images and reference locally.

Comment (absheth) [reply web-ui/src/components/Profile/profile.jsx]: I agree with @bhushanmalgaonkar

**Comment by absheth** [comment]
Merging this pull request so that I can pull and start working on the login and registration page

File: web-ui/src/components/Profile/profile.jsx (hunk 1 above)
Comment (chetan253) [reply web-ui/src/components/Profile/profile.jsx]: I'll fix it in next commit

**Pull request merged**